docs/source/classes.rst

```diff
@@ -16,35 +16,33 @@ The subject of a **Pedigree** is represented by an **Individual** class. This cl
    :header-rows: 1
 
    * - Field
-     - Type
      - Multiplicity
      - Description
    * - id
-     - ID
      - 1..1
-     - Logical ID
-   * - name
-     - string
-     - 0..1
-     - Name of the individual
-   * - identifiers
-     - array of ``Identifier``
-     - 0..*
-     - External identifiers for the individual
+     - External identifier for the individual
    * - sex
-     - ``Concept``
      - 1..1
-     - Sex assigned at birth, values: ``male``, ``female``, ``other``, ``unknown``; See `Gender Harmony Context Definitions <https://confluence.hl7.org/display/VOC/Gender+Harmony+Context+Definitions>`_ and `FHIR AdministrativeGender Value Set <http://hl7.org/fhir/ValueSet/administrative-gender>`_.
+     - Sex assigned at birth
    * - gender
-     - ``Concept``
      - 0..1
-     - Presumed or reported gender identity, values: ``male``, ``female``, ``non-binary``, ``non-disclosed``, ``trans``, etc.; See `Gender Harmony Context Definitions <https://confluence.hl7.org/display/VOC/Gender+Harmony+Context+Definitions>`_.
-   * - lifeStatus
-     - ``Concept``
+     - Presumed or reported gender identity
+   * - name
+     - 0..1
+     - Name of the individual
+   * - dateOfBirth
+     - 0..1
+     - Birth date of the individual, can be just birth year in most cases
+   * - age
+     - 0..1
+     - Age of the individual
```

Comment:
I think this explicitly needs to be able to represent age, age range, or gestational age (Since we've removed lifeStatus which had the unborn option. We need to represent pregancies.)

Reply:
@buske , so on the common dataset document, I saw there are three age-related columns: 1) Age, 2) Estimated Age, and 3) Age Range. As you commented out, since we've deprecated `lifeStatus`, adding Gestational Age makes sense.

My question is, do we want to spell out all these three (or four) attributes on the Individual class?

Reply:
@liberaliscomputing I think that as long as we explicitly mention gestational age in the description, that's likely good for now.

Reply:
Hi @mbaudis, thanks for the comment! Actually, `estimatedAge` is a `boolean` element annotating whether the given `age` is estimated or not.